typings/mysql/index.d.ts

```diff
@@ -53,25 +53,51 @@ export function createPool(config: PoolOptions): BasePool;
 
 export function createPoolCluster(config?: PoolClusterOptions): PoolCluster;
 
-export function escape(value: any): string;
-
-export function escapeId(value: any): string;
-
+/**
+ * @file index.d.ts
+ * 
+ * @author marvinhagemeister
+ * @date 11 July 2017
+ * 
+ * @dependency \@types/sqlstring
+ * @see {@link https://www.npmjs.com/package/@types/sqlstring} for more information
+ */
+export function escape(value: any, stringifyObjects?: boolean, timeZone?: 'local' | NonNullable<unknown>): string;
+/**
+ * @file index.d.ts
+ * 
+ * @author marvinhagemeister
+ * @date 11 July 2017
+ * 
+ * @dependency \@types/sqlstring
+ * @see {@link https://www.npmjs.com/package/@types/sqlstring} for more information
+ */
+export function escapeId(value: any, forbidQualified?: boolean): string;
+/**
+ * @file index.d.ts
+ * 
+ * @author marvinhagemeister
+ * @date 11 July 2017
+ * 
+ * @dependency \@types/sqlstring
+ * @see {@link https://www.npmjs.com/package/@types/sqlstring} for more information
+ */
 export function format(sql: string): string;
 export function format(
   sql: string,
-  values: any[],
-  stringifyObjects?: boolean,
-  timeZone?: string,
-): string;
-
-export function format(
-  sql: string,
-  values: any,
+  values: any | any[],
   stringifyObjects?: boolean,
-  timeZone?: string,
+  timeZone?: 'local' | NonNullable<unknown>,
 ): string;
-
+/**
+ * @file index.d.ts
```

Comment:
@ScrapsPrograms, I was thinking something simple like:

> - Top of the file ⬇️

```ts
/**
 * sqlstring typings were taken from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/sqlstring/index.d.ts
 */

// ...
```

Or "based" instead of "taken", since the typings aren't the same 🙋🏻‍♂️